cves/CVE-2016-2162.yml

```diff
@@ -119,19 +127,22 @@ discovered:
     The "apache" flag can be true, false, or nil.
     If there is no evidence as to how this vulnerability was found, then you may
     leave the entries blank except for "answer". Write down where you looked in "answer".
-  answer:
+  answer: |
+    I looked through every link from the CVE tracker, and also searched for the
+    reporter, the CVE, and for any timeline associated with it. Not much came up
+    that was not already on the initial CVE page (or linked pages).
   date:
   automated:
-  apache:
-  contest:
+  apache: false
```

Comment:
Found by Paolo Perliti (see https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=62687301).  However, I also could not find the date of discovery, only date of publish.